src/node_report.cc

```diff
@@ -52,6 +52,9 @@ extern char** environ;
 # include <sys/param.h>     // MAXHOSTNAMELEN on Linux and the BSDs.
 #endif  // __POSIX__
 
+#define NANOS_PER_SEC 1000 * 1000 * 1000
```

Comment:
Can you put parens around the expression and remove them on line 507?

Reply:
Or use a constexpr?